### DISCUSSION TITLE
IPv6 website access support

### DISCUSSION THREAD

**Discussion by cardosocristian** [forem/forem#17573] [Feature Requests]
Hi
An interesting feature that would be very useful to help with the growth of the internet is for the site to support IPv6 access, in several regions of the world IPv4 is already scarce and systems do not support IPv6 access, generating a series of complex mechanisms tunneling to access services that are IPv4-only.

**Comment by theoparis** [comment]
I noticed https://dev.to does not have an AAAA record either. Can ipv6 support for dev.to be considered in the future?
